vllm/v1/spec_decode/utils.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+from vllm.v1.sample.metadata import SamplingMetadata
+
+
+def is_spec_decode_supported(req_idx: int,
+                             sampling_metadata: SamplingMetadata) -> bool:
+    if ((sampling_metadata.top_p and sampling_metadata.top_p[req_idx] < 1.0) or
+        (sampling_metadata.top_k and sampling_metadata.top_k[req_idx] > 0)):
```

Comment:
Isn't top_k == vocab_size by default?

Reply:
yes, it is. I changed it to 
`0 < sampling_metadata.top_k[req_idx] < vocab_size`